nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/computation/log/ComputationRunner.java

```diff
@@ -321,6 +321,8 @@ protected void checkpointIfNecessary() {
                 checkpoint();
                 completed = true;
                 inCheckpointRecords = inRecords;
+            } catch (Exception e) {
+                log.error(e, e);
```

Comment:
Is this intended ? It could hide problems as the current runner won't be stopped.

Reply:
Bad move actually, thanks !